src/gridfs/download.ts

```diff
@@ -237,28 +243,24 @@ function doRead(stream: GridFSBucketReadStream): void {
     let errmsg: string;
     if (doc.n > expectedN) {
       errmsg = `ChunkIsMissing: Got unexpected n: ${doc.n}, expected: ${expectedN}`;
-      return __handleError(stream, new MongoDriverError(errmsg));
+      return __handleError(stream, new MongoGridFSChunkError(errmsg));
     }
 
     if (doc.n < expectedN) {
       errmsg = `ExtraChunk: Got unexpected n: ${doc.n}, expected: ${expectedN}`;
-      return __handleError(stream, new MongoDriverError(errmsg));
+      return __handleError(stream, new MongoGridFSChunkError(errmsg));
     }
 
     let buf = Buffer.isBuffer(doc.data) ? doc.data : doc.data.buffer;
 
     if (buf.byteLength !== expectedLength) {
       if (bytesRemaining <= 0) {
         errmsg = `ExtraChunk: Got unexpected n: ${doc.n}`;
-        return __handleError(stream, new MongoDriverError(errmsg));
+        return __handleError(stream, new MongoGridFSChunkError(errmsg));
       }
 
-      errmsg =
-        'ChunkIsWrongSize: Got unexpected length: ' +
-        buf.byteLength +
-        ', expected: ' +
-        expectedLength;
-      return __handleError(stream, new MongoDriverError(errmsg));
+      errmsg = `ChunkIsWrongSize: Got unexpected length: ${buf.byteLength}, expected: ${expectedLength}`;
+      return __handleError(stream, new MongoGridFSChunkError(errmsg));
```

Comment:
__handleError is a oneliner (maybe it contained more at some point) can we just inline it everywhere?

```js
      return stream.emit(
        GridFSBucketReadStream.ERROR,
        new MongoGridFSChunkError(
          `ChunkIsWrongSize: Got unexpected length: ${buf.byteLength}, expected: ${expectedLength}`
        )
      );
```